vaadin-testbench-core/src/main/java/com/vaadin/testbench/unit/ComponentWrap.java

```diff
@@ -21,7 +22,7 @@
  * wrappers.
  *
  * @param <T>
- *         component type
+ *            component type
```

Comment:
Seems there is a difference in our formatters. Could fix this so the un-needed moving front and back doesn't happen.

Reply:
I'm using Eclipse Code Formatter within IntelliJ IDEA with Flow repo rules (eclipse/VaadinJavaConventions.xml).
Perhaps there's still something wrong on my side.
I'll check and fix it, thanks.